### ISSUE TITLE
[Suggestion] Weapons made of modded materials

### ISSUE THREAD

**Issue by Sunconure11** [Draylar/gate-of-babylon#77]
If this is within your grasp, perhaps you could try and add some compat to such mods, and give their materials weapons here.

**Comment by Lynxanor** [comment]
yeah id love compatibility with mods like better nether and better end, great mod btw

**Comment by ghost** [comment]
dev has given up. you'll need to find another mod.
